docs/operators/aws_cloudwatch_input.md

```diff
@@ -0,0 +1,126 @@
+## `aws_cloudwatch_input` operator
+
+The `aws_cloudwatch_input` operator reads logs from AWS Cloudwatch Logs using [AWS's SDK](https://github.com/aws/aws-sdk-go)
+
+The `aws_cloudwatch_input` operator will use the `Timestamp` field of the event as the parsed entry's timestamp. All other fields are added to the entry's record.
+
+The `aws_cloudwatch_input` operator will use the following order to get credentials. Environment Variables (Details [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html)), Shared Credentials file (Details [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html)), Shared Configuration file (if SharedConfig is enabled details [here](https://docs.aws.amazon.com/sdkref/latest/guide/creds-config-files.html)) , and EC2 Instance Metadata (credentials only details [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-metadata.html)). You can provide `profile` to specify which credential set to use from a Shared Credentials file.
+
+### Configuration Fields
+
+| Field                     | Default                | Description                                                                                                 |
+| ---                       | ---                    | ---                                                                                                         |
+| `id`                      | `aws_cloudwatch_input` | A unique identifier for the operator.                                                                       |
+| `output`                  | Next in pipeline       | The connected operator(s) that will receive all outbound entries.                                           |
+| `log_group_name`          | required               | The Cloudwatch Logs Log Group Name.                                                                         |
+| `region`                  | required               | The AWS Region to be used.                                                                                  |
+| `log_stream_name_prefix`  |                        | The log stream name prefix to use. This will find any log stream name in the group with the starting prefix |
+| `log_stream_names`        |                        | An array of log stream names to get events from.                                                            |
```

Comment:
It seems it would not be inline with what we have done in the past. Looking over other docs it seems that we do not indicate they are "optional". The parameters are only called out when "required".